kinto/core/resource/schema.py

```diff
@@ -150,3 +151,71 @@ class BookmarkSchema(ResourceSchema):
 
     def preparer(self, appstruct):
         return strip_whitespace(appstruct)
+
+
+class CSVQuerystring(colander.SchemaNode):
+    """String field representing a list of attributes."""
+
+    schema_type = colander.String
+    missing = colander.drop
+    error_message = "The value should be a list of comma separated field names"
+
+    def deserialize(self, cstruct=colander.null):
+        params = super(CSVQuerystring, self).deserialize(cstruct)
+        if params is colander.drop:
+            return params
+        else:
+            return params.split(',')
```

Comment:
This can definitely be improved but I didn't understand how to use `preparer` to do it. Thinking through it, maybe defining it as `Sequence` we can do something as simple as:

```python
class FieldList(colander.SchemaNode):
    fields = colander.SchemaNode(colander.String(), missing=colander.drop)

    def deserialize(self, cstruct=colander.null):
        if isinstance(cstruct, six.string_types):
            cstruct = cstruct.split(',')
        return super(FieldList, self).deserialize(cstruct)

```